packages/embedded-registration/src/constants.ts

```diff
@@ -1 +1,3 @@
 export const EMAIL_SPAM = 'SPAM';
+
+export const UTM_COOKIES = ['utm_source', 'utm_campaign'];
```

Comment:
Could we please instead of hardcoding that values - allow to specify it as Registration _config_ arguments with default value as  `['utm_source', 'utm_campaign']`. Thanks to this in the future marketing team could independently set list of URL params to track.

Reply:
Sure, much better idea